p2p/host/basic/obsaddr_test.go

```diff
@@ -44,62 +40,62 @@ func newConn(local, remote ma.Multiaddr) *mockConn {
 	return &mockConn{local: local, remote: remote}
 }
 
-func normalize(addr ma.Multiaddr) ma.Multiaddr {
-	for {
-		out, last := ma.SplitLast(addr)
-		if last == nil {
-			return addr
-		}
-		if _, err := last.ValueForProtocol(ma.P_CERTHASH); err != nil {
```

Comment:
Could we just remove the certhash and everything after as the normalize function? We might need to be careful around circuit addresses.

That would remove the dependency on the webtransport/webrtc transports